libraries/AP_Motors/AP_MotorsHeli_Single.cpp

```diff
@@ -196,6 +196,9 @@ void AP_MotorsHeli_Single::set_update_rate( uint16_t speed_hz )
 // init_outputs - initialise Servo/PWM ranges and endpoints
 void AP_MotorsHeli_Single::init_outputs()
 {
+
+    bool init_ok = _frame_class == MOTOR_FRAME_HELI;
```

Comment:
This needs to be after the if on line 202. Otherwise that init will never be done.